packages/js/settings-editor/src/legacy/content.tsx

```diff
@@ -36,15 +37,22 @@ const Form = ( { settings }: { settings: SettingsField[] } ) => {
 export const LegacyContent = ( {
 	settingsPage,
 	activeSection,
+	settingsData,
 }: {
 	settingsPage: SettingsPage;
 	activeSection: string;
+	settingsData: SettingsData;
 } ) => {
 	const section = settingsPage.sections[ activeSection ];
 
 	if ( ! section ) {
 		return null;
 	}
 
-	return <Form settings={ section.settings } />;
+	return (
+		<>
+			<CustomView html={ settingsData.start.content } />
```

Comment:
I think this is blocking build https://github.com/woocommerce/woocommerce/actions/runs/13065656219/job/36457497092?pr=54948
```
Error: ../../packages/js/settings-editor build:project:cjs: src/legacy/content.tsx(54,23): error TS18048: 'settingsData.start' is possibly 'undefined'.
```

Reply:
@psealock

Reply:
Thanks @ilyasfoo, fixed in https://github.com/woocommerce/woocommerce/pull/54948/commits/343d947b5d8ec6bbaa1c63817072628afc272932